### PR TITLE
Failing link check

### PR DESCRIPTION
**Pull request by regularfry**
Just a sanity check to see what the behaviour of the link checker is around redirects.

### CODE REVIEW

**Comment by regularfry** [comment]
Setting "ready for review" to get the markdown link checker action to run.  Don't worry about reviewing.

File: .github/workflows/markdown-check-links.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
-  "aliveStatusCodes": [200, 403]
+  "aliveStatusCodes": [200, 403],
```
Comment (stefaniuk) [inline .github/workflows/markdown-check-links.json:2]: Do we need to introduce the `403` HTTP status as alive?

**Review by stefaniuk — APPROVED** [review]
LGTM

**Comment by regularfry** [comment]
Ok, so this has passed.  Looking through the dependency stack I couldn't see anything obviously wrong about the way timeouts are handled, but replacing redirected links with their eventual destination means fewer network requests, so fewer opportunities to go wrong. The increased timeout also means that if we are getting rate-limited at the other end, we're less likely to see a failure as a result.

**Comment by andyblundell** [comment]
Ah is there a timeout setting on this action - it does throw a few false negatives, which are a nuisance

**Comment by regularfry** [comment]
> Ah is there a timeout setting on this action - it does throw a few false negatives, which are a nuisance

Yeah, I've changed it to 30 seconds.  It defaults to 10.  I'm unclear what would fail at 10 seconds but pass at 30. It just doesn't seem like a very natural breakpoint unless we're getting rate limited somewhere.

**Comment by walteck** [comment]
These updates have made it into main through other branches - I'd propose that we close this PR and branch. @regularfry - any objection?

**Comment by regularfry** [comment]
Agreed.

**Pull request closed without merging**